Dockerfile

```diff
@@ -13,12 +13,18 @@ LABEL repository="https://github.com/shenxianpeng/cpp-linter-action"
 LABEL maintainer="shenxianpeng <20297606+shenxianpeng@users.noreply.github.com>"
 
 RUN apt-get update
-RUN apt-get -y install curl jq
+RUN apt-get -y install python3-pip
+RUN python3 -m pip install --upgrade pip
+# curl jq
 
-COPY runchecks.sh /entrypoint.sh
-RUN chmod +x /entrypoint.sh
+# COPY runchecks.sh /entrypoint.sh
+# RUN chmod +x /entrypoint.sh
+COPY python_action/ /pkg/python_action/
+COPY setup.py /pkg/setup.py
+RUN python3 -m pip install /pkg/
```

Comment:
The command here seems can not install the package from requirements.txt, maybe need to run `python3 -m pip install -r /pkg/requirements.txt` to install. but it's OK, we can fix it after merge if it is a problem.

Reply:
_requirements.txt_ isn't used in the install process. I believe I added that file for building docs (I have forgotten why I added the _requirements.txt_). Look in the _setup.py_ file, under the `setup()` you'll see a `install_requires` parameter.

Reply:
I remember now. I added the requirements file because I needed to install the dependencies only (not the entire pkg) when running pylint.

Reply:
Got it.